### ISSUE TITLE
HLSLTokenizer prb

### ISSUE THREAD

**Issue by dgu123** [unknownworlds/hlslparser#2]
Dear team ,

during the compilation , next error happend,
==== Building HLSLParser (debug) ====
HLSLTokenizer.cpp
In file included from ../src/HLSLTokenizer.cpp:4:0:
../src/HLSLTokenizer.h:92:61: error: ‘size_t’ has not been declared
     HLSLTokenizer(const char\* fileName, const char\* buffer, size_t length);
                                                             ^
../src/HLSLTokenizer.cpp:92:1: error: prototype for ‘M4::HLSLTokenizer::HLSLTokenizer(const char_, const char_, size_t)’ does not match any in class ‘M4::HLSLTokenizer’
 HLSLTokenizer::HLSLTokenizer(const char\* fileName, const char\* buffer, size_t length)
 ^
In file included from ../src/HLSLTokenizer.cpp:4:0:
../src/HLSLTokenizer.h:83:7: error: candidates are: M4::HLSLTokenizer::HLSLTokenizer(const M4::HLSLTokenizer&)
 class HLSLTokenizer
       ^
../src/HLSLTokenizer.h:92:5: error:                 M4::HLSLTokenizer::HLSLTokenizer(const char_, const char_, int)
     HLSLTokenizer(const char\* fileName, const char\* buffer, size_t length);
     ^
make[1]: **\* [obj/Debug/HLSLTokenizer.o] Erreur 1

**Comment by zz85** [comment]
looks was the issue i'm was referring to in #1. you need to add `#include <stddef.h>`

**Comment by dgu123** [comment]
Yes it help  but then i needed to add #include <string.h> #include <limits.h> in string.cpp
regards
david
